code/modules/mob_spawn/corpses/mob_corpses.dm

```diff
@@ -42,12 +42,24 @@
 	outfit = /datum/outfit/syndicatecommandocorpse/lessenedgear
 
 /datum/outfit/syndicatecommandocorpse/lessenedgear
-	name = "Syndicate Commando Corpse"
+	name = "Syndicate Commando Corpse (Less Antag Gear)"
 	gloves = /obj/item/clothing/gloves/tackler
 	back = null
 	id = null
 	id_trim = null
 
+/obj/effect/mob_spawn/corpse/human/syndicatecommando/soft_suit
+	outfit = /datum/outfit/syndicatecommandocorpse/soft_suit
+
+/datum/outfit/syndicatecommandocorpse/soft_suit
```

Comment:
wouldnt it make sense to make this a subtype of line 44 since the point is less gamer gear